postgresql/scale-postgresql-server/scale-postgresql-server.sh

```diff
@@ -14,7 +14,7 @@ az postgres server create \
 --location westus \
 --admin-user myadmin \
 --admin-password <server_admin_password> \
---sku-name GP_Gen4_2 \
+--sku-name GP_Gen5_2 \
 
 # Monitor usage metrics - CPU
 az monitor metrics list \
```

Comment:
L21, L27:

__Required__: Subscription ID is used in multiple places in this script and is not clear that it's a user-supplied value. Make it a variable assigned by the user.

Similarly, the resource group name, location, and server name should be user-supplied values.